jenkins_wrapper.sh

```diff
@@ -57,15 +57,17 @@ export LSSTSW=${LSSTSW:-$WORKSPACE/lsstsw}
 
   OPTS=()
 
-  if [[ $python == "py3" ]]; then
+  # shellcheck disable=SC2154
```

Comment:
So where is `$python` coming from. Is it an environment variable? Can you add a comment on this disable line to explain.